### PR TITLE
Add example with global DATA variable.

### PR DESCRIPTION
**Pull request by majjoha**
The global `DATA` variable in Ruby can be incredibly handy for various operations such as extracting information from a large piece of text. I've added a new TIL that demonstrates its abilities.

I hope you'll find it useful.

### CODE REVIEW

**Comment by mike-burns** [comment]
Oh hey, we listen to the same music.

(No further critique from me.)

**Comment by zamith** [comment]
This is great. TIL.

File: ruby/global-data-variable.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+# Global `DATA` variable
```
Comment (JoelQ) [inline ruby/global-data-variable.md]: `DATA` is a constant, not a global variable. Thoughts on changing the title to something along the lines of: `# The DATA constant`?

File: ruby/global-data-variable.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+The global `DATA` variable in Ruby allows us to access the text at the end of
```
Comment (JoelQ) [inline ruby/global-data-variable.md]: This should say "The `DATA` constant in Ruby ..."

**Comment by majjoha** [comment]
I've updated the PR based on your comments, @JoelQ. I hope it looks better this time.

**Comment by JoelQ** [comment]
Looks good! Thanks for sharing @majjoha . I've merged this as: 3237f70123c3dfa837be380c8384f647c56ceec5

**Pull request closed without merging**